templates/gulpfile.js

```diff
@@ -0,0 +1,23 @@
+const gulp = require("gulp");
+const nearUtils = require("near-runtime-ts");
```

Comment:
let's have separate package for utils instead of bundling it into `near-runtime-ts` (e.g. `near-bindgen-ts` or `near-asc-utils` if it's not only about bindings generation). I think it's confusing for same module to fit different purposes when it is required from JS-side. When I require something advertised as runtime library, I don't expect to get tools which help me use compiler.

Reply:
also I think that we still need gulpfile as we'll likely need to build another "wallet contract" / "wallet app" .wasm file. I'll have NEP soon, see #product-wallet in Discord if interested in general idea.